tidy3d/components/data/sim_data.py

```diff
@@ -191,15 +250,15 @@ def _get_scalar_field(
         """
 
         if field_name[0] == "S":
-            dataset = self.get_poynting_vector(field_monitor_name)
+            dataset = self._get_poynting_vector(field_monitor_data)
```

Comment:
@caseyflex the test fails because it asserts that `_get_scalar_field()` calls `get_poynting_vector` and therefore errors because this method raises a  `NotImplementedError` in `JaxSimulationData`.

You can safely comment out the last two lines in the adjoint test

```py
    with pytest.raises(NotImplementedError):
        sim_data._get_scalar_field(mnt_name_differentiable, "S", "abs")
```

I think they are a bit unnecessary.

Reply:
thanks!